code/game/gamemodes/game_mode.dm

```diff
@@ -80,11 +80,17 @@
 		report = config.intercept
 	addtimer(CALLBACK(GLOBAL_PROC, .proc/display_roundstart_logout_report), ROUNDSTART_LOGOUT_REPORT_TIME)
 
-	SSblackbox.set_details("round_start","[time2text(world.realtime)]")
-	if(SSticker && SSticker.mode)
-		SSblackbox.set_details("game_mode","[SSticker.mode]")
-	if(GLOB.revdata.commit)
-		SSblackbox.set_details("revision","[GLOB.revdata.commit]")
+	if(SSdbcore.Connect())
+		var/sql
+		if(SSticker && SSticker.mode)
+			sql += "game_mode = '[SSticker.mode]'"
+		if(sql)
+			sql += ", "
+		if(GLOB.revdata.commit)
+			sql += "commit_hash = '[GLOB.revdata.commit]'"

```

Comment:
You can do that in a later pr, this is designed to mirror the data already going in to the feedback table, so it should be the same data.